queries/date.py

```diff
@@ -136,307 +143,7 @@ def help_text(lemma: str) -> str:
-    'bolludagur:kk'_et/fall
-
-QDateHalloween/fall →
-    'hrekkjavaka:kvk'_et/fall
-    | "halloween"
-
-QDateSovereigntyDay/fall →
-    'fullveldisdagur:kk'_et/fall
-
-QDateFirstDayOfSummer/fall →
-    'sumardagur:kk'_et_gr/fall 'fyrstur:lo'_et_kk/fall
-    | 'fyrstur:lo'_et_kk/fall 'sumardagur:kk'_et/fall
-
-QDateThorlaksMass/fall →
-    'Þorláksmessa:kvk'_et/fall
-
-QDateChristmasEve/fall →
-    'aðfangadagur:kk'_et/fall 'jól:hk'_ef?
-
-QDateChristmasDay/fall →
-    'jól:hk'/fall
-    | 'jóladagur:kk'_et/fall
-
-QDateNewYearsEve/fall →
-    'gamlárskvöld:hk'_et/fall
-    | 'gamlársdagur:kk'_et/fall
-
-QDateNewYearsDay/fall →
-    'nýársdagur:kk'_et/fall
-
-QDateNewYear/fall →
-    'áramót:hk'_ft/fall
-
-QDateWorkersDay/fall →
-    'baráttudagur:kk'_et/fall 'verkalýður:kk'_et_ef
-
-QDateEaster/fall →
-    'páskar:kk'/fall
-
-QDateEasterSundayWord_nf -> "páskasunnudagur"
-QDateEasterSundayWord_þf -> "páskasunnudag"
-QDateEasterSundayWord_þgf -> "páskasunnudegi"
-QDateEasterSundayWord_ef -> "páskasunnudags"
-
-QDateEasterSunday/fall →
-    'páskadagur:kk'_et/fall
-    | QDateEasterSundayWord/fall
-
-QDateMaundyThursday/fall →
-    'skírdagur:kk'_et/fall
-
-QDateGoodFriday/fall →
-    'föstudagur:kk'_et_gr/fall 'langur:lo'_et_kk/fall
-
-QDateNationalDay/fall →
-    'þjóðhátíðardagur:kk'_et/fall
-    | 'þjóðhátíðardagur:kk'_et/fall 'Íslendingur:kk'_ft_ef
-    | 'þjóðhátíðardagur:kk'_et/fall 'Ísland:hk'_et_ef
-    | 'þjóðhátíð:kvk'_et/fall 'Íslendingur:kk'_ft_ef
-    | 'þjóðhátíð:kvk'_et/fall 'Ísland:hk'_et_ef
-
-QDateBankHoliday/fall →
-    'verslunarmannahelgi:kvk'_et/fall
-    | 'frídagur:kk'_et/fall 'verslunarmaður:kk'_ft_ef
-
-QDateCultureNight/fall →
-    'menningarnótt:kvk'_et/fall
-
-QDateValentinesDay/fall →
-    'Valentínusardagur:kk'_et/fall
-    | 'Dagur'/fall 'elskandi:kk'_ft_ef
-    | 'dagur:kk'_et/fall 'elskandi:kk'_ft_ef
-
-QDateMansDay/fall →
-    'bóndadagur:kk'_et/fall
-
-QDateWomansDay/fall →
-    'konudagur:kk'_et/fall
-
-QDateMardiGrasDay/fall →
-    'sprengidagur:kk'_et/fall
-    | 'sprengikvöld:hk'_et/fall
-
-QDatePalmSunday/fall →
-    'pálmasunnudagur:kk'_et/fall
-
-QDateMothersDay/fall →
-    'mæðradagur:kk'_et/fall
-
-QDateSeamensDay/fall →
-    'sjómannadagur:kk'_et/fall
-
-QDateFirstDayOfWinter/fall →
-    'fyrstur:lo'_et_kk/fall 'vetrardagur:kk'_et/fall
-    | 'vetrardagur:kk'_et_gr/fall 'fyrstur:lo'_et_kk/fall
-
-QDateFathersDay/fall →
-    'feðra-dagur:kk'_et/fall  # Hack because 'feðradagur' isn't in BÍN
-    | 'feðradagur:kk'_et/fall
-
-QDateIcelandicTongueDay/fall →
-    'dagur:kk'_et/fall "íslenskrar" "tungu"
-    | 'dagur:kk'_et/fall "íslenskrar" 'Tunga'_ef_kvk
-    | 'Dagur'/fall "íslenskrar" "tungu"
-    | 'Dagur'/fall "íslenskrar" 'Tunga'_ef_kvk
-
-QDateSecondChristmasDay/fall →
-    'annar:lo'_et_kk/fall "í" "jólum"
-
-QDateSummerSolstice/fall →
-    'sumarsólstöður:kvk'_ft/fall
-
-QDateWinterSolstice/fall →
-    'vetrarsólstöður:kvk'_ft/fall
-
-$score(+55) QDate
-
-"""
+GRAMMAR = read_grammar_file("date")
```

Comment:
I don't know if this is a good idea, but maybe we should support automatically loading grammar file with same name as the file where read_grammar_file fn is called? Something like this:

https://stackoverflow.com/questions/59004195/python-how-to-get-the-absolute-path-of-file-where-a-function-was-called#59004672

But I don't know. Explicit is better than implicit acc. Zen of Python.

Reply:
We could call the function with something like `__name__` as an argument I guess?

Reply:
I think it's probably best to be explicit in this case and just provide the filename like so `read_grammar_file("module")`.